cmd/consul_exporter/consul_exporter.go

```diff
@@ -53,6 +53,7 @@ func main() {
 		healthSummary = kingpin.Flag("consul.health-summary", "Generate a health summary for each service instance. Needs n+1 queries to collect all information.").Default("true").Bool()
 		kvPrefix      = kingpin.Flag("kv.prefix", "Prefix from which to expose key/value pairs.").Default("").String()
 		kvFilter      = kingpin.Flag("kv.filter", "Regex that determines which keys to expose.").Default(".*").String()
+		metaFilter    = kingpin.Flag("meta.filter", "Regex that determines which meta key/values to expose.").Default("").String()
```

Comment:
This appears to only match on the key names, not the values.

Rather than make the default empty string for "don't match any", you could make the default `^$` so that there are no key matches. That would simplify the loop below.

Reply:
I'm indifferent to this change - with the empty string it allows us to cut out the loop entirely, although certainly in my case it's not really going to make a meaningful difference to resource usage so I've accepted your suggestion.